nix/solc-static.nix

```diff
@@ -40,7 +40,8 @@ stdenv.mkDerivation rec {
     sha256 = "${sha256}";
   };
 
-  nativeBuildInputs = [ autoPatchelfHook makeWrapper ];
+  nativeBuildInputs = [ makeWrapper ]
+    ++ lib.optional stdenv.isLinux autoPatchelfHook;
```

Comment:
why?

Reply:
the build failed for me, it tried to patch the macos binary